awx/ui/src/screens/Inventory/ConstructedInventory.js

```diff
@@ -20,9 +20,9 @@ import JobList from 'components/JobList';
 import RelatedTemplateList from 'components/RelatedTemplateList';
 import { ResourceAccessList } from 'components/ResourceAccessList';
 import RoutedTabs from 'components/RoutedTabs';
-import ConstructedInventoryDetail from './ConstructedInventoryDetail';
+import ConstructedInventoryDetail from './InventoryDetail';
```

Comment:
We should change this back to `ConstructedInventoryDetail`. The constructed view will need to make additional requests to fetch the details, it will sync over websockets, and it will have RBAC rules based on the inventory and inventory source. I believe it is different enough to deserve its own Detail component.